docusaurus/docs/React/guides/theming/input-ui.mdx

```diff
@@ -3,213 +3,810 @@ id: input_ui
 title: Message Input UI
 ---
 
-## How-to Guide for Building a Custom Message Input
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
 
-The React Chat component library provides a highly customizable `MessageInput` component. We'll outline the various ways in which you
-can customize the look and behavior of this component, ranging from simple style changes, all the way to creating a completely
-new input with custom logic.
+## How To Build Your Own Message Input
```

Comment:
```suggestion
```
I think we can get rid of this title as __UI COOKBOOK__ category insinuates that we're going to be building things.